frontend/src/components/Dragable/DragPanel/DragPanel.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import "./DragPanel.scss";
+import DragableField from "../DragableField/DragableField";
+
+const generatePanel = () => {
+  const digits = [1, 2, 3, 4, 5, 6, 7, 8, 9];
+  const rows = digits.map((val, idx) => {
+    return (
+      <tr key={idx}>
+        <td key={idx}>
+          <DragableField value={val} />
+        </td>
+      </tr>
+    );
+  });
+  return (
+    <table className="dragPanel">
+      <tbody>{rows}</tbody>
+    </table>
+  );
+};
+
+export default function DragPanel() {
+  return (
+    <div className="background">
+      <div> {generatePanel()} </div>
```

Comment:
Unnecessary div. If you need any properties from div you should apply it to the `dragPanel` css class.